addons/medical_engine/CfgMoves.hpp

```diff
@@ -8,6 +8,76 @@ class CfgMovesMaleSdr: CfgMovesBasic {
             weaponIK = 0;
         };
 
+        class Unconscious;
+        class DeadState;
+        class ace_unc_1: DeadState {
```

Comment:
It's written with full word literally everywhere but this class and file name, might as well do it here, would be a lot more readable, I had no clue what "unc" was at first.

Reply:
just config names or also file names?

Reply:
Files renamed and configs names changed